src/views/iam/RolePermissionTab.vue

```diff
@@ -227,6 +232,47 @@ export default {
         this.resetNewFields()
         this.fetchData()
       })
+    },
+    csv ({ data = null, columnDelimiter = ',', lineDelimiter = '\n' }) {
```

Comment:
can we have a better name for method, something like generateRulesCSV, etc

Reply:
> can we have a better name for method, something like generateRulesCSV, etc

updated method name